package.nls.json

```diff
@@ -78,6 +78,8 @@
     "AWS.explorerNode.registry.error": "Error loading registry schema items",
     "AWS.explorerNode.registry.noSchemas": "[No Registry Schemas]",
     "AWS.explorerNode.registry.registryName.Not.Found": "Registry name not found",
+    "AWS.explorerNode.connecting.label": "Connecting...",
+    "AWS.explorerNode.connecting.tooltip": "Connecting...",
```

Comment:
Eliminate one of these unless you think it'll be localized differently.

Reply:
Maintaining separate strings reduces the risk of updating the text with the intention of only changing the text in one of the two locations.

Reply:
I don't like that justification. We've been trying to move towards more generic strings. Use one for now and create a second string if they diverge.